src/store/index.ts

```diff
@@ -1,13 +0,0 @@
-import { combineReducers } from 'redux';
```

Comment:
ce fichier n'est pls necessaire ? par quoi a-t-il ete remplacé ?

Reply:
ce qui est fait dans ce fichier est condensé dans la fonction configureStore (cf investigation). C'est remplacé par le fichier store/store.ts
pour plus d'infos : https://redux.js.org/usage/migrating-to-modern-redux#store-setup-with-configurestore